client/src/components/TagMenu/TagMenu.component.tsx

```diff
@@ -77,7 +80,114 @@ const TagMenu = (): ReactElement => {
       )
     : useQuery(FETCH_TAGS_QUERY_KEY, () => fetchTags())
 
-  // TODO - create an AccordionItem for agency tags
+  const isShowMobileVariant = useBreakpointValue({
+    base: true,
+    xs: true,
+    sm: false,
+    md: false,
+    lg: false,
+    xl: false,
+  })
+
+  const TagMenuMobile = tags && (
+    <VStack align="left" spacing={0}>
+      {tags
+        .filter(
+          ({ tagType, tagname }) =>
+            tagType === TagType.Topic && tagname !== queryState,
+        )
+        .sort((a, b) => (a.tagname > b.tagname ? 1 : -1))
+        .map((tag) => {
+          const { tagType, tagname } = tag
```

Comment:
```suggestion
        .map(({ id, tagType, tagname }) => {
```
then replace all references to tag with the appropriate property